src/PrestaShopBundle/Controller/ArgumentResolver/SearchParametersResolver.php

```diff
@@ -102,9 +102,11 @@ public function resolve(Request $request, ArgumentMetadata $argument)
     {
         $filtersClass = $argument->getType();
         list($controller, $action) = ControllerAction::fromString($request->get('_controller'));
-        // is the url contains filters?
+
         $query = $request->query;
-        if ($query->has('filters') || $query->has('limit')) {
+        $doesTheUrlContainsFilters = ($query->has('filters') || $query->has('limit') || $query->has('sortOrder'));
```

Comment:
No need to add this in variable, can be used in condition =)

Reply:
The variable name is a hint to help developers to understand the meaning of this line of code. It replaces the [deleted comment](https://github.com/PrestaShop/PrestaShop/pull/9300/files#diff-307533e1154f5cf404d67226aa79ef8eL105).